linux-pull-run.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+docker pull ghcr.io/badgerloop-software/chase-car-dashboard-image:latest
+[[ $OSTYPE != 'darwin'* ]] && xdg-open http://localhost:4000
+[[ $OSTYPE == 'darwin'* ]] && open http://localhost:4000
+docker run -p 4000:3000 ghcr.io/badgerloop-software/chase-car-dashboard-image:latest
+
```

Comment:
For windows too - 2 things:
1. Perhaps add a flag to disable automatic opening of the web browser in case someone wants to use one they already have open
2. Consider adding help message to print the URL of the dashboard in case open or xdg-open does not work

Reply:
updated